firmware/quadruna/BMS/src/jobs.c

```diff
@@ -9,6 +9,7 @@
 #include "app_commitInfo.h"
 #include "app_heartbeatMonitors.h"
 #include "app_stackWaterMarks.h"
+#include "app_tractiveSystem.h"
```

Comment:
It would be good practice to indicate what dependencies from the header files is being used in a comment